source/backend/entities/PimsBaseContext.cs

```diff
@@ -1698,6 +1704,38 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             entity.Property(e => e.EffectiveDateHist).HasDefaultValueSql("(getutcdate())");
         });
 
+        modelBuilder.Entity<PimsConsultationOutcomeType>(entity =>
+        {
+            entity.HasKey(e => e.ConsultationOutcomeTypeCode).HasName("OUTCMT_PK");
+
+            entity.ToTable("PIMS_CONSULTATION_OUTCOME_TYPE", tb =>
+                {
+                    tb.HasComment("Description of the consultation outcome type for a lease or license.");
+                    tb.HasTrigger("PIMS_OUTCMT_I_S_I_TR");
+                    tb.HasTrigger("PIMS_OUTCMT_I_S_U_TR");
```

Comment:
why is the context being updated?

Reply:
Oh, this was branched from before the scaffold was merged, I'll rebase.